pkg/operator/controllers/guardrails/staticresources/gk_audit_controller_deployment.yaml

```diff
@@ -100,8 +100,8 @@ spec:
             - ALL
           readOnlyRootFilesystem: true
           runAsNonRoot: true
-          seccompProfile:
-            type: RuntimeDefault
+          # seccompProfile: #intended deviation from upstream
```

Comment:
Why are we deviating from upstream? Do we have anything that explains our reasoning any what we gain vs what we risk? This looks low risk, but I want to make sure.

Reply:
well, these is a "long" story regarding this, tl;dr is with this enabled, gatekeeper does not start on 4.11 and onwards. more details can be found in  #3074, which is the initial fix for the problem. 
and after some more investigation, this PR came up as a better solution, which applies more strict SCCs then the upstream requires on the gatekeeper roles.

Reply:
and this deviation is actually suggested by gatekeeper in order to run on openshift 4.x: https://open-policy-agent.github.io/gatekeeper/website/docs/v3.11.x/vendor-specific/#running-on-openshift-4x

Reply:
Thanks this is really helpful!